1/style.css

```
@@ -0,0 +1,103 @@
+body {
+  background-color: black;
+  color: white;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+footer {
+  bottom: 0;
+  padding-bottom: 10px;
+  position: fixed;
+  width: 100%;
+  text-align: center;
+}
+input {
+  width: 96%;
+}
+button {
+  width: 97.5%;
+  background-color: aquamarine;
+}
+
+#title {
+  display: flex;
+  justify-content: center;
+}
+#foot {
+  display: flex;
+  justify-content: center;
+}
+#form {
+  margin-top: 3px;
+  display: flex;
+  flex-direction: column;
+  padding-top: 2vh;
+}
+.subtitle {
+  margin: 0px;
+  display: flex;
+  justify-content: center;
+  padding-top: 1vh;
+}
+
+#box {
+  margin-left: 250px;
+  border: 3px white;
+  height: 30vh;
+  width: 29vw;
+  background-color: rgb(225, 143, 250);
+  padding-left: 1vw;
+  border-radius: 25px;
+  box-shadow: 0 0 0 1px rgba(252, 251, 251, 0.5),
+    0 0 10px 10px rgba(244, 238, 238, 0.2);
+  transition: 0.75s;
+}
+
+#box:hover {
+  height: 33vh;
+  width: 31vw;
+}
+
+#box2 {
+  margin-left: 250px;
+  border: 3px white;
+  height: 20vh;
+  width: 29vw;
+  background-color: rgb(225, 143, 250);
+  padding-left: 1vw;
+  border-radius: 25px;
+  box-shadow: 0 0 0 1px rgba(252, 251, 251, 0.5),
+    0 0 10px 10px rgba(244, 238, 238, 0.2);
+  transition: 0.75s;
+}
+
+#box2:hover {
+  height: 22vh;
+  width: 31vw;
+}
+
+#order {
+  padding-top: 15vh;
+  display: flex;
+  align-items: center;
+}
+
+.space {
+  padding-bottom: 1vh;
+}
+.space2 {
+  padding-bottom: 2vh;
+}
+
+.iput {
```

Acho que falta aqui um `n`
```suggestion
.input {
```